stl/inc/bitset

```diff
@@ -400,7 +400,18 @@ public:
     }
 
     _NODISCARD bool all() const noexcept {
-        return count() == size();
+        if _CONSTEXPR_IF (_Bits == 0) {
+            return true;
+        }
+        for (size_t _Wpos = 0; _Wpos < _Words; ++_Wpos) {
+            if (~_Array[_Wpos] != 0) {

```

Comment:
Wouldn't this be better? (Same on line 412)
```suggestion
            if (_Array[_Wpos] != ~static_cast<_Ty>(0)) {
```

Reply:
Yes. Surprisingly, output a bit changes. Here's new: https://godbolt.org/z/nWzmae